firmware/cmake/embedded.cmake

```diff
@@ -184,6 +184,7 @@ function(embedded_binary
 )
     message("  ➕ [embedded.cmake, embedded_binary()] Creating Embedded Target for ${BIN_NAME}")
     set(ELF_NAME "${BIN_NAME}.elf")
+    set_source_files_properties(${BIN_SRCS} PROPERTIES COMPILE_FLAGS "-fsanitize=undefined")
```

Comment:
I don't like this but target properties will leak into all the interface libraries. I want to just target user code, which I is exclusively touched here